### DISCUSSION TITLE
Easier means of removing a folder from a nested hierarchy

### DISCUSSION THREAD

**Discussion by swhi1106** [standardnotes/forum#2662] [Suggestions] [answered]
**Describe the issue**
A clear and concise description of what the issue is.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by swhi1106** [comment]
Once a tag is indented under a "folder" I can't seem to find a way to undent it.   Too easy to indent accidently with drag and drop

**Comment by effieeee** [comment, marked as answer]
Hi @swhi1106, you'll have to drag the folder to the bottom most portion of your tag list where there will be a corresponding space to remove it from the hierarchy. We'll try to keep it in mind to have an easier way of doing this 🙏
